### PR TITLE
Feat: Home

### PR DESCRIPTION
**Pull request by sri-rishi**
### Folders to check 

- component
- home

### CODE REVIEW

File: src/component/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    AiOutlineTwitter,
+    MdAlternateEmail
+} from "../../assets";
+
+export const Footer = () => {
+    return (
+        <footer className="footer-box flex-column justify-center align-center gap-2">
+            <p>Made in <BsCodeSlash  className="icon-vr-align "/> by Rishi</p>
+            <div className="footer-icons flex-row align-center gap-3">
+                <a className="social-icon" href="https://github.com/sri-rishi/nakshatra-watch-ecomm">
```
Comment (Ankur9669) [inline src/component/Footer/Footer.jsx:14]: You can replace a tags with Link

File: src/component/Navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.nav-div {
+    position: fixed;
+    top: 0;
+    z-index: 9999;
```
Comment (Ankur9669) [inline src/component/Navbar/navbar.css:4]: You can use variables for z-index

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Footer, Navbar } from "../../component/index"
+import { CategoriesSection } from "./components/categorySection/CategorySection"
+import { HeroSection } from "./components/HeroSection"
+import { PopularQuizzes } from "./components/popularQuizzesSection/PopularSection"
+
+export const Home = () => {
+    return (
+        <div>
+            <Navbar />
+            <main class="main-div flex-column justify-center">
```
Comment (Ankur9669) [inline src/pages/Home/Home.jsx:10]: change class to className

**Review by Ankur9669 — APPROVED** [review]
The UI looks nice, you just need to replace a tags with Link rest seems fine, Great work

**Review by Naman-Saxena1 — APPROVED** [review]
Apart from points suggested by Ankur, wasn't able to find any issues, code looks fine and UI is really neat, nice work 👍

File: src/component/Footer/Footer.jsx (hunk 1 above)
Comment (sri-rishi) [reply src/component/Footer/Footer.jsx:14]: they are outer link, that's why I have used a tags

File: src/pages/Home/Home.jsx (hunk 3 above)
Comment (sri-rishi) [reply src/pages/Home/Home.jsx]: Done

**Pull request merged**